docs/getting-started-with-scalardb.md

```diff
@@ -235,35 +170,67 @@ public class ElectronicMoneyWithTransaction extends ElectronicMoney {
     }
   }
 
-  @Override
+  public int getBalance(String id) throws TransactionException {
+    // Start a transaction
+    DistributedTransaction tx = manager.start();
+
+    try {
+      // Retrieve the current balances for id
+      Get get =
+              Get.newBuilder()
+                      .namespace(NAMESPACE)
+                      .table(TABLENAME)
+                      .partitionKey(Key.ofText(ID, id))
+                      .build();
+      Optional<Result> result = tx.get(get);
+
+      int balance = -1;
+      if (result.isPresent()) {
+        balance = result.get().getInt(BALANCE);
+      }
+
+      // Commit the transaction
+      tx.commit();
+
+      return balance;
+    } catch (Exception e) {
+      tx.abort();
+      throw e;
+    }
+  }
+
   public void close() {
     manager.close();
   }
 }
 ```
 
-As you can see, it's not very different from the code with `DistributedStorage`.
-This code instead uses `DistributedTransactionManager` and all the CRUD operations are done through the `DistributedTransaction` object returned from `DistributedTransactionManager.start()`.
+Now we can run the application.
 
-Now let's run the application with transaction mode.
+- Charge `1000` to `user1`:
+```shell
+$ ./gradlew run --args="-action charge -amount 1000 -to user1"
 ```
-$ ../../gradlew run --args="-mode transaction -action charge -amount 1000 -to user1"
-$ ../../gradlew run --args="-mode transaction -action charge -amount 0 -to merchant1"
-$ ../../gradlew run --args="-mode transaction -action pay -amount 100 -to merchant1 -from user1"
-```
-
-## Use JDBC transaction
 
-When you use a JDBC database as a backend database, you can optionally use the native transaction manager of a JDBC database instead of the default `ConsensusCommit` transaction manager.
-
-To use the native transaction manager, you need to set `jdbc` to a transaction manager type in **scalardb.properties** as follows.
+- Charge `0` to `merchant1` (Just create an account for `merchant1`):
+```shell
+$ ./gradlew run --args="-action charge -amount 0 -to merchant1"
+```
 
+- Pay `100` from `user1` to `merchant1`:
+```shell
+$ ./gradlew run --args="-action pay -amount 100 -from user1 -to merchant1"
 ```
-scalar.db.transaction_manager=jdbc
+
+- Get the balance of `user1`:
+```shell
+$ ./gradlew run --args="-action getBalance -id user1"
 ```
 
-You don't need to set a key `transaction` to `true` in Scalar DB schema for the native transaction manager.
-So you can use the same schema file as **emoney-storage.json**.
+- Get the balance of `merchant1`:
+```shell
+$ ./gradlew run --args="-action getBalance -id merchant1"
+```
```

Comment:
Also added the `getBalance` action that's a read only transaction to check the balance of a specified user.